app/views/pages/guidance.html.erb

```diff
@@ -70,7 +70,7 @@
       <p class="govuk-body">They’ll also be updated in UCAS Apply within two hours.</p>
 
       <h4 class="govuk-heading-s">2.</h4>
-      <p class="govuk-body">To add a new location, click the green ‘Add a location’ button on the Locations page. Note that you can only enter 37 locations in total. </p>
+      <p class="govuk-body">To add a new location, click the green ‘Add a location’ button on the Locations page. </p>
```

Comment:
This is the displayed from the transcript for the video, perhaps that would need editing?

Reply:
https://www.youtube.com/watch?v=W_zhfemKiYM